File: javascript/bubble_sort/src/bubble_sort.js
const sort = (arr) => {
    let swap = true;
    while(swap) {
        swap = false;
        for (let i = 0; i < arr.length - 1; i++) {
            if ( arr[i + 1] < arr[i] ) {
                let temp = arr[i + 1];
                arr[i + 1] = arr[i];
                arr[i] = temp;
                swap = true;
            }
        }
    }
    return arr;
}

exports.sort = sort;

File: python/bubble_sort/bubble_sort.py
def sort(lst):
    swap = True
    while swap:
        swap = False
        for i in range(len(lst) - 1):
            if lst[i + 1] < lst[i]:
                temp = lst[i]
                lst[i] = lst[i + 1]
                lst[i + 1] = temp
                swap = True
    return lst
